streaming/src/main/scala/org/apache/spark/streaming/dstream/FileInputDStream.scala

```diff
@@ -74,10 +74,12 @@ class FileInputDStream[K, V, F <: NewInputFormat[K,V]](
     directory: String,
     filter: Path => Boolean = FileInputDStream.defaultFilter,
     newFilesOnly: Boolean = true,
-    conf: Option[Configuration] = None)
+    @transient conf: Option[Configuration] = None)
```

Comment:
I believe it has to be `private` or Scala will make a getter/setter and thus make it a member too. You can double-check by using `javap` or even the byte code plugin in IJ.